zerver/lib/markdown/help_settings_links.py

```diff
@@ -15,6 +15,11 @@
 
 REGEXP = re.compile(r"\{settings_tab\|(?P<setting_identifier>.*?)\}")
 
+
+# If any changes to this link mapping are made,
```

Comment:
Just a trick for this sort of comment: If you have it start with `TODO/compatibility`, our process includes scanning for any of those to clean up after every major release.

Reply:
I guess in this case, we'll delete the whole file once we're done, so that's not necessary here.